parsec/backend/invite.py

```diff
@@ -191,7 +193,7 @@ def generate_invite_email(
 ) -> Message:
     mail_text = MAIL_TEXT[email_config.language]
     parsec_url = "https://parsec.cloud/get-parsec"
-    invite_link = str(
+    parsec_invite_link = str(
```

Comment:
Better to use the `BackendInvitationAddr.to_url` method than `str` conversion (it's kind of more explicit)